webapp/src/pages/offer/components/OfferComponent.js

```diff
@@ -147,6 +164,28 @@ class Offer extends Component {
                   : "Not available"}
               </div>
             </div>
+
+            <div class="row pb-5 ">
+              <div class="col-6 font-weight-bold pr-4"  align="right">
+                <div class="form-check accommodation-container">
```

Comment:
This should only show if they've been awarded accomodation.
Is that the case? 

Same with travel_award